pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -2,7 +2,7 @@
 , version
 , meta
 , updateScript ? null
-, binaryName ? "firefox"
+, binaryName ? pname
```

Comment:
I'm opposed to this change. 

It does not hold true for all consumers and the default serves as an example of how the values look.